.github/workflows/mypy_check.yml

```diff
@@ -0,0 +1,15 @@
+name: "mypy check"
+on: [pull_request]
```

Comment:
Can we run this on develop too? That helps just in case 2 PRs get merged in quick succession and their changes have clashing types

Reply:
Yes, we can run it on develop.

Reply:
I think you need to change the workflow so that it runs on develop. For example, here's how we do it in oppia/oppia:

https://github.com/oppia/oppia/blob/24298204cfaf7ade44fcf7542418cc1d66de77a1/.github/workflows/backend_tests.yml#L5-L12

Reply:
I have made the changes, it now runs on develop and thank you for the resource.

Reply:
Hmm I think now it runs on pull requests opened against `develop`, but I think you still need the `push` key to tell it to run whenever a commit gets pushed to develop (i.e. whenever a PR merges). You can read GitHub's documentation here: https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-events-to-trigger-workflows

Reply:
Thank you for the resource. I have added the `push` event.